### PR TITLE
support file attack

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Fix issue #39 

TODO: update the document: https://github.com/chaos-mesh/website/issues/249

```bash
./bin/chaosd attack file --help
File attack related commands

Usage:
  chaosd attack file [command]

Available Commands:
  append      append file
  create      create file
  delete      delete file
  modify      modify file privilege
  rename      rename file
  replace     replace data in file

Flags:
  -h, --help   help for file

Global Flags:
      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
      --uid string         the experiment ID

Use "chaosd attack file [command] --help" for more information about a command.
```

### CODE REVIEW

**Comment by cwen0** [comment]
@WangXiangUSTC Why need `FileTool` tool?

**Comment by WangXiangUSTC** [comment]
> @WangXiangUSTC Why need `FileTool` tool?

We can involve `FileTool` into `chaos-daemon` later, and then `chaos-daemon` just need to call `FileTool`, avoid to writing the same code again. @cwen0

File: test/integration_test/file/run.sh [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+${bin_path}/chaosd attack file append --file-name ${chaos_test_file} --data "chaos-mesh" --count 5 --uid 12345
+
+num=`cat ${chaos_test_file} | grep "chaos-mesh" | wc -l`
+if [[ ${num} -ne 5 ]]; then
+    echo "append file failed"
+    exit 1
+fi
+
+${bin_path}/chaosd recover 12345
+num=`cat ${chaos_test_file} | grep "chaos-mesh" | wc -l`
```
[the remark this answers is not in the document]
Comment (WangXiangUSTC) [reply test/integration_test/file/run.sh:84]: ignore

File: test/integration_test/file/run.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
```
[the remark this answers is not in the document]
Comment (WangXiangUSTC) [reply test/integration_test/file/run.sh:19]: ignore

**Review by cwen0 — APPROVED** [review]
LGTM

File: tools/file/replace.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	"fmt"
+	"os/exec"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+func NewFileReplaceCommand() *cobra.Command {
```
Comment (Andrewmatilde) [inline tools/file/replace.go:26]: I think the REPLACE command here is complex to use.

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**